### DISCUSSION TITLE
技术预览版 - 7

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#325] [通知]
## 🚀 功能

- 支持切换弹幕控制器可见性 @Richasy (#318)
- 提供BBDown下载集成 @Richasy (#317)

## 🐛 Bug 修复

- 添加FLV备选播放 @Richasy (#324)
- 修复预启动设置未保存的问题 @Richasy (#316)
- 修复全窗口模式下的白角问题 @Richasy (#313)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.2801.pre-release'>技术预览版 - 7</a>.</em>

**Comment by Richasy** [comment]
感谢 @nilaoda 的帮助，哔哩现在可以通过BBDown下载视频

**Reply by X0980** [reply]
媒体控件优化了，牛啊牛
